icefoganalytics/elcc-data-management
# Tooling: Add Development Instructions

# Context

Add development instructions to the readme.
Add some edge case handling for users missing their first and/or last name.

# Testing Instructions

1. `git clone git@github.com:icefoganalytics/elcc-data-management.git` to a new location
2. Follow the setup instructions and see if you can boot the app successfully.
